skipper.go

```diff
@@ -2028,6 +2025,20 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 	return listenAndServeQuit(o.CustomHttpHandlerWrap(proxy), &o, sig, idleConnsCH, mtr, cr)
 }
 
+func initializeTracer(o Options) (ot.Tracer, error) {
```

Comment:
Lets make it a method similar to `filterRegistry()`

Reply:
Makes sense, thanks